pkg/kubelet/kubelet.go

```diff
@@ -1615,6 +1625,30 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 	}
 }
 
+// syncStatus syncs pods statuses with the apiserver.
+func (kl *Kubelet) syncStatus() {
+	glog.V(3).Infof("Syncing pods status")
+
+	for _, pod := range kl.pods {
+		status, err := kl.GetPodStatus(GetPodFullName(&pod), pod.UID)
+		if err != nil {
+			glog.Warningf("Error getting pod %s status: %v, retry later", pod.Name, err)
```

Comment:
nit: I tend to always use %q for strings which prints out the string in quotes. It makes it clearer what's in the string and catches errors like empty string and whitespace characters. Feel free to heed or ignore :)